src/backend/decorators/resource/resource-decorator.ts

```diff
@@ -318,6 +318,9 @@ class ResourceDecorator {
       editProperties: this.getProperties({
         where: 'edit',
       }).map((property) => property.toJSON('edit')),
+      newProperties: this.getProperties({
+        where: 'new',
```

Comment:
You might have to extend the type here:
https://github.com/SoftwareBrothers/adminjs/blob/master/src/frontend/interfaces/property-json/property-json.interface.ts#L3

Thanks for contributing; I'll try to take a proper look at this PR next week.

Reply:
done ;)